app/src/main/java/com/example/aaa/pinkcalculator/MainActivity.java

```diff
@@ -75,202 +76,189 @@ private void nextOperation() {
     }
 
     public void onClick(View v) {
-
         switch (v.getId()) {
-            case R.id.b_clickingOn_0:
+            case R.id.null_button:
                 nextOperation();
```

Comment:
I see the same block of code and you use a different value for `.append();` function.

I suggest you first find append value inside `switch-case` and after that make `append` operation